### DISCUSSION TITLE
is anyone successfuly configure codeium?

### DISCUSSION THREAD

**Discussion by twmht** [NvChad/NvChad#2686] [General]
I have installed codeium(https://github.com/Exafunction/codeium.vim
) and configure it to cmp source. it works as it's integrated to cmp.

Here is what i configured (https://github.com/aquilesg/nvchadconfig/blob/main/configs/overrides.lua#L82)

however, the experence is bad because i can't achieve the effect as shown in the demo gif (https://user-images.githubusercontent.com/1908017/213154744-984b73de-9873-4b85-998f-799d92b28eec.gif)

and i can't generate the code based on the comment i write above.

is there anyone successfuly configure codeium to work like the demo with nvchad?

**Comment by siduck** [comment]
```lua
  {
        "hrsh7th/nvim-cmp",
        config = function(_, opts)
            table.insert(opts.sources, { name = "codeium" })
            require("cmp").setup(opts)
        end,

        dependencies = {
            {
                "jcdickinson/codeium.nvim",
                config = function()
                    require("codeium").setup({})
                end,
            },
        },
    },
